### ISSUE TITLE
[release/2.8] timm_regnet got fail_accuracy

### ISSUE THREAD

**Issue by kaileiyx** [intel/torch-xpu-ops#1814]
### 🐛 Describe the bug

timm_regnet got fail_accuracy with PT2.8

Reproducer:
python benchmarks\dynamo\torchbench.py --accuracy -d xpu -n10 --backend=inductor --cold-start-latency --training  --amp-dtype float16  --only timm_regnet   

error:
loading model: 0it [00:00, ?it/s]
loading model: 0it [00:06, ?it/s]
xpu  train timm_regnet                        
E0702 09:26:48.458000 516443 torch/_dynamo/utils.py:2998] RMSE (res-fp64): 0.00426, (ref-fp64): 0.00131 and shape=torch.Size([]). res.dtype: torch.float32, multiplier: 3.000000, tol: 0.001000, use_larger_multiplier_for_smaller_tensor: 0
fail_accuracy

### Versions

pytorch: pip install --pre torch==2.8.0 --index-url https://download.pytorch.org/whl/test/xpu
triton: pytorch-triton-xpu-3.4.0
torch-xpu-ops: https://github.com/intel/torch-xpu-ops/commit/3a9419c8bb6a98dd3e3cd473c36691fb4abeae40
OS:Ubuntu 24.10
Machine: BMG
